### ISSUE TITLE
Error on load (with 5.10 release, after syncing back and forth a bit with sync.automerge.org) - error inflating document chunk ops: an operation referenced a missing actor id

### ISSUE THREAD

**Issue by heckj** [automerge/automerge#919]
I don't have a solid reproduction of this issue - I was working on debugging the sync and connections for automerge-repo-swift (using the demo app MeetingNotes) when I encountered this. The libraries were all recently updated to the 0.5.10 core library release, not sure of the state of `sync.automerge.org`. I'm suspecting there was an issue in the flight back and forth.

It appeared when I loaded a binary after debugging sessions, and the one on fisk started throwing this error:

```
error inflating document chunk ops: an operation referenced a missing actor id
```

I'm attaching the raw file (as compressed/ZIP version since GitHub it cranky about file types) for inspection:

[rawautomergefile.zip](https://github.com/automerge/automerge/files/15227475/rawautomergefile.zip)

**Comment by heckj** [comment]
duplicate of #897 - with an example file to show the result.

**Comment by alexjg** [comment]
This issue should be fixed in `0.5.11`. I'm going to close this but please reopen if it occurs again.
